parsing-scripts/create_maintainers_fixtures.py

```diff
@@ -0,0 +1,103 @@
+import json
+
+with open('portindex.json', "r") as file:
+    data = json.load(file)
```

Comment:
You should dynamically populate maintainers while reading portindex json. Read a port, check if maintainer exists; create one if it doesn't exist; else just find the maintainer's id and enter the relation port-maintainer in the additional table.

Reply:
Yes, I have changed it to do the same. I will update the pull request once I am done with today's schedule.